DuggaSys/analytic.js

```diff
@@ -1268,6 +1269,7 @@ function loadUserInformation(){
 				localStorage.setItem('analyticsLastUser', $(this).val());
 			} catch(err) { }
 
+			selectPage.change();
```

Comment:
The following row is executed a couple of times each second and causes after a while a browser crash. I would probably not blame this PR for the cause of this infinite loop, however with this this addition the site becomes unusable after a while. 
This is a console log on the row just under this addition to the codebase:
![image](https://user-images.githubusercontent.com/49141837/82987174-69dc3880-9ff7-11ea-9f56-963b9c7dc729.png)

I believe this is also the cause of the issue in the following GIF that makes the dropdown not function after switching pages a couple of times. (My clicks are outlined with the black circle.)
![userinfo](https://user-images.githubusercontent.com/49141837/82987272-8e381500-9ff7-11ea-8d84-8e185bb810b9.gif)